stdlib/3/tempfile.pyi

```diff
@@ -17,6 +18,11 @@ TMP_MAX: int
 tempdir: Optional[str]
 template: str
 
+if sys.version_info >= (3, 6):
+    _DirT = Union[AnyStr, os.PathLike[AnyStr]]
```

Comment:
`AnyStr` is a type variable, and thus you can't use it in an alias. I believe you will need to use something like `Union[str, bytes, os.PathLike[Union[str, bytes]]]`

Reply:
In that case it unfortunately seems that the best solution is indeed to duplicate all signatures.